code/game/mecha/working/clarke.dm

```diff
@@ -48,24 +48,6 @@
 		var/mob/living/brain/B = M.brainmob
 		hud.add_hud_to(B)
 
-/obj/mecha/working/clarke/domove(direction)
-	. = ..()
-	if (.)
-		if(prob(30))
-			crush_floor()
-
-/obj/mecha/working/clarke/mechturn(direction)
-	if(prob(80))
-		crush_floor()
-	return ..()
-
-///Damages floor tiles and plating as the Clarke moves over them with its tracks. Damage is superficial, will not create hull breaches.
```

Comment:
this would suggest it is not intentional that this proc is suppose to create hull breaches.

Reply:
I haven't seen it cause breaches, but it does break plating, which is more annoying to replace.